lib/heroku_rails_deploy/deployer.rb

```diff
@@ -153,8 +150,11 @@ def run_heroku_command(app_name, command, validate: nil)
         # If we're running a shell command, return the underlying
         # shell command exit code
         cli_command << ' --exit-code'
+        display_output = true
```

Comment:
Agreed. I originally plumbed the `display_output` option through the different methods to run heroku commands, but when I looked at where I would actually use the option it seems cleaner to tie it the "run" subcommands.